### ISSUE TITLE
Recurrent Dropout in LSTM

### ISSUE THREAD

**Issue by logen1004** [facebookarchive/caffe2#1048]
Is there a plan to support more powerful dropoutwrapper in LSTM. Like in TensorFlow and Keras, it can do input, output, recurrent dropout, while the DropoutCell only supply output dropout.
